handlers/main.yml

```diff
@@ -1,6 +1,7 @@
 # handlers file for supervisor
 ---
 - name: restart supervisor
+  become: yes
```

Comment:
When I do it at the playbook level I have the error I've been posting, as
if the handler didn't get it...
On Jul 5, 2016 6:09 PM, "Mischa ter Smitten" <notifications@github.com>
wrote:

> In handlers/main.yml
> <https://github.com/Oefenweb/ansible-supervisor/pull/20#discussion_r69529691>
> :
>
> > @@ -1,6 +1,7 @@
> >  # handlers file for supervisor
> >  ---
> >  - name: restart supervisor
> > +  become: yes
>
> This does not work in Ansible < 2.
>
> And I don't see why this could not be set the at playbook level?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Oefenweb/ansible-supervisor/pull/20/files/e783aa138321ff1db78f59bfdd141d8e3036d339#r69529691>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANgSFKKIdqMWezutNdVxoq97cbQuORVks5qSh9UgaJpZM4JE391>
> .
>